src/pyclts/transcriptionsystem.py

```diff
@@ -207,7 +207,9 @@ def _parse(self, string):
                         string, sound1, sound2, self)
                 elif sound1.type == 'consonant' and \
                         sound1.manner in ('stop', 'implosive', 'click', 'nasal') and \
-                        sound2.manner in ('stop', 'implosive', 'affricate', 'fricative'):
+                        sound2.manner in (
```

Comment:
YOu have not verified the results on the whole of PYclts data, the modifications you propose are intrusive to the code and wil result in tons of additional clusters. I therefore close the PR.